packages/calypso-e2e/src/element-helper.ts

```diff
@@ -90,18 +88,21 @@ export async function clickNavTab(
 	const navTabItem = page.locator( selectors.navTabItem( { name: name, selected: false } ) );
 	await Promise.all( [ page.waitForNavigation(), navTabItem.click() ] );
 
-	// Final verification.
+	// Final verification, check that we are now on the expected
+	// navtab.
 	const newSelectedTabLocator = page.locator(
 		selectors.navTabItem( { name: name, selected: true } )
 	);
 	const newSelectedTabName = await newSelectedTabLocator.innerText();
 
-	// Strip numerals from the extracted tab name, similar to above.
 	if ( newSelectedTabName.replace( /[0-9]|,/g, '' ) !== name ) {
 		throw new Error(
 			`Failed to confirm NavTab is active: expected ${ name }, got ${ newSelectedTabName }`
 		);
 	}
+
+	// Wait for all network activity to cease.
+	await page.waitForLoadState( 'networkidle', { timeout: 20 * 1000 } );
```

Comment:
**Suggestion (semi-blocking):** Does the script still work without this? I did some more poking around, and from what I can tell, people seem to agree that networkidle waits apart from a document load can be risky. Is there a specific request we can wait for? I think we should find some way to take this out if possible!

Context:
[this comment](https://github.com/microsoft/playwright/issues/2515#issuecomment-724163391) and [this comment](https://github.com/microsoft/playwright/issues/2526#issuecomment-642223890)

Reply:
This is a difficult one.

I'll start by answering the question on whether it works or not. It **does work**, but I don't like it either.

This method `ElementHelper.clickNavtab` has to support Atomic and Simple. We both know from experience that Simple sites have the more problematic `networkidle` behaviour, as Simple sites appear to make network requests well into the 15-20 second range after the page loads. 

Because this method is meant to be generic the only request it can potentially wait for must be on both types of sites and present on any page in Calypso that has a navtab involved. This is a pretty tall order and I haven't identified anything that meets all of the requirements. 

I can do either one of the following (while I wait for the Treehouse request to clear):
1. merge this PR as-is and take a deeper dive into this problem in another PR.
2. iterate on this one since your comment is semi-blocking.